src/sentry/buffer/redis.py

```diff
@@ -229,15 +275,24 @@ def _process_single_incr(self, key):
                 self.logger.debug('buffer.revoked.empty', extra={'redis_key': key})
                 return
 
-            model = import_string(values['m'])
-            filters = pickle.loads(values['f'])
+            model = import_string(values.pop('m'))
+            if values['f'].startswith('{'):
```

Comment:
Unsure to me if it's possible for pickle data to start with a `{` (or `[` byte from below). I can't find if this would map up to any opcode, but this is a bit risky, since `(` is a valid starting byte, for example. So we're just real close to one that may be.

Reply:
I dont THINK it is (I think all data is type prefixed).

Reply:
Well, I mean, `(` is valid. I just can't find any types that would trigger `{` to be one though.

```python
>>> pickle.dumps([]).startswith('(')
True
```